content/en/notebooks/_index.md

```diff
@@ -78,6 +75,40 @@ To copy the URL for a specific cell, click the cell's **Share** menu and select
 
 When a user visits the URL for a specific cell, the notebook is opened to show the cell at the top of the viewport. Links are absolute. A cell's URL remains the same even if it is moved to a new position within the notebook.
 
+## Add images to notebooks
+
+<div class="alert alert-info">Only PNG, JPG, JPEG, and GIF file types are supported. Uploads have a max file size of 4MB.</a></div>
+
+You can add images to your notebook using either the [image cell](#image-cell) or the  [markdown editor](#markdown-editor).
+
+### Image cell
+
+This approach places the image in a separate cell from your text, but provides options for resizing and justifying the image as well as adding a caption. Images uploaded by the Image cell are hosted by Datadog.
```

Comment:
```suggestion
This approach places the image in a separate cell from your text, and provides options for resizing, aligning, and captioning the image. Images uploaded by the image cell are hosted by Datadog.
```

- changed `but` to `and`—`but` suggests a contradiction of an expectation, which i don't think is the case here
- parallel construction for a sentence that's otherwise a bit long and meandering
- changed `justifying` to `aligning`—i think aligning is more immediately clear, whereas justifying an image can also mean "giving a good reason to have this image"